Sources/NostrSDK/Events/DirectMessageEvent.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DirectMessageEvent.swift
+//  
+//
+//  Created by Joel Klabo on 8/10/23.
+//
+
+import Foundation
+
+/// An event that contains an encrypted message
+///
+/// > Note: [NIP-04 Specification](https://github.com/nostr-protocol/nips/blob/master/04.md)
+public final class DirectMessageEvent: NostrEvent, DirectMessageEncrypting {
+
+    public func decryptedContent(keypair: Keypair) throws -> String {
```

Comment:
1. Since in this function you only need the keypair's private key, let's only require that as the argument (the need-to-know principle).
2. Could we make the function more readable at the call site? Something like
```swift
public func decryptedContent(using privateKey: PrivateKey) throws -> String
```